src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3849,23 +3849,34 @@
      *
      * <p>For example:</p>
      * <pre>
-     *         String hello = """
-     *                 Hello, world!
-     *                 """;
+     * public class MyClass {
```

Comment:
You can use old example, but use corrected subtree representation in `parses as:` section

Reply:
done

Reply:
Not done, I meant use old example without class definition
```
     *         String hello = """
     *                 Hello, world!
     *                 """;
```

Reply:
done